pycaret/classification/functional.py

```diff
@@ -859,6 +905,12 @@ def create_model(
         Score grid is not printed when verbose is set to False.
 
 
+    engine: Optional[str] = None
+        The engine to use for the model, e.g. for lr, users can
+        switch between "sklearn" and "sklearnex" by specifying
+        engine="sklearnex".
```

Comment:
```suggestion
    engine: Optional[str] = None
        The execution engine to use for the model, e.g. for Logistic Regression ("lr"), users can
        switch between "sklearn" and "sklearnex" by specifying
        `engine="sklearnex"`.
```

Please update in other places